strawberryfields/gbs/data.py

```diff
@@ -54,16 +58,26 @@
 | **MUTAG_2** | **MUTAG_3** |
 +-------------+-------------+
 
+Molecules
+---------
+
+Using the :mod:`~.gbs.vibronic` module, GBS data has been generated for formic acid at zero

```

Comment:
How about also mentioning the `sample.vibronic` function?

Reply:
Will add this.